# SPDX-FileCopyrightText: Huawei Inc.
#
# SPDX-License-Identifier: Apache-2.0

Align compiler warning arguments with gcc.

Apply to repo third_party/googletest

Signed-off-by: Esben Haabendal <esben.haabendal@huawei.com>
Upstream-Status: Pending

diff --git a/BUILD.gn b/BUILD.gn
index fa6eea4fb5d0..e27c070a0b85 100755
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -84,6 +84,7 @@ config("gmock_private_config") {
 config("gmock_config") {
   include_dirs = [ "googlemock/include" ]
 
+  if (is_clang) {
   cflags_cc = [
     # The MOCK_METHODn() macros do not specify "override", which triggers this
     # warning in users: "error: 'Method' overrides a member function but is not
@@ -92,6 +93,7 @@ config("gmock_config") {
     # fixed.
     "-Wno-inconsistent-missing-override",
   ]
+  }
 }
 
 static_library("gmock") {
